_health-care/_js/_components/date_input.jsx

```diff
@@ -43,7 +43,7 @@ class DateInput extends React.Component {
   render() {
     const error_class = this.state.hasError ? "usa-input-error" : ""
     return (
-      <div className={`usa-input-grid usa-input-grid-medium ${error_class}`}>
+      <div className={"usa-input-grid usa-input-grid-medium ${error_class}"}>
```

Comment:
Oops, thought this was meant to be quotation marks, used to Ruby string interpolation. I will correct this. And yes the spec was failing, so converting back to backticks fixes it.